ilagan_andreijireh/script.js

```diff
@@ -128,4 +127,111 @@ function searchCountry() {
       document.getElementById('countryDetails').innerHTML = '<p>An error occurred: ' + error.message + '</p>';
       document.getElementById('sameRegionCountries').innerHTML = '';
     });
-}
\ No newline at end of file
+}
+
+document.addEventListener("DOMContentLoaded", function () {
+  loadArtists();
+});
+
+async function loadArtists() {
+  try {
+    const response = await fetch('https://your-api-domain.com/api/opm-artists');
```

Comment:
Good day, Andrei!

I am just wondering if where is this connected? "Your-api-domain" is this really the name of your domain?